Python/bytecodes.c

```diff
@@ -4009,6 +4028,7 @@ dummy_func(
 
         op(_EXIT_TRACE, (--)) {
             TIER_TWO_ONLY
+            frame->instr_ptr = CURRENT_TARGET() + _PyCode_CODE(_PyFrame_GetCode(frame));
```

Comment:
If you want to, you can revert this change (plus the ones in `ceval.c` and `ceval_macros.h`) since `_FOR_ITER_TIER_TWO` has been reworked.